internal/pkg/config/fleet.go

```diff
@@ -67,12 +67,12 @@ func strToLevel(s string) (zerolog.Level, error) {
 		l = zerolog.DebugLevel
 	case "info":
 		l = zerolog.InfoLevel
-	case "warning":
+	case "warn":
```

Comment:
Are there backwards compatibility concerns with this?

I suppose there aren't since agent is the only thing that can start Fleet server, and the versions are always in sync. I would feel better about not forgetting something if we just accepted "warning" and "warn", but maybe I am just being paranoid.

Reply:
Thanks!

Reply:
Already changed to accept both values